src/test/java/io/aiven/klaw/service/ClusterApiServiceTest.java

```diff
@@ -50,9 +50,9 @@ public class ClusterApiServiceTest {
 
   private UtilMethods utilMethods;
 
-  @Mock HandleDbRequests handleDbRequests;
+  @Mock HandleDbRequestsJdbc handleDbRequests;
 
-  @Mock MailUtils mailService;
+  @Mock MailUtils çƒç;
```

Comment:
probably it's better to use letters only from english ABC
or is that anything I probably missed?

Reply:
Actually didn't need this declaration. Probably some type.